### PR TITLE
Responsive design for layout

### PR DESCRIPTION
**Pull request by garenpham**
Things I have done:
1. (fixing #26) Change the "Transfer Status" text to "Send Tokens". Added a new route '/send-tokens' in ClientRoutes.js, as well as test for that route in ClientRoutes.test.js.
2. Regarding #29:
- Implement a new design for the Side Menu.
- Side Menu now shows only icons initially, and the user can press the Menu Button to view more details about the section name.
- Change the logo Greenstand to be in the middle for mobile layout, on desktop it stays the same (at the left).
- Add (*).test.js file to each new component.
image
image
image
image

### CODE REVIEW

File: src/components/Routes/ClientRoutes.js [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
-import Wallet from "../../pages/Wallet/Wallet";
-import TransferStatus from "../../pages/TransferStatus/TransferStatus";
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import SendTokens from '../../pages/SendTokens/SendTokens';
+import TransferStatus from '../../pages/TransferStatus/TransferStatus';
+import Wallet from '../../pages/Wallet/Wallet';
 
 const ClientRoutes = () => {
-  return (
```
Comment (OlhaD) [inline src/components/Routes/ClientRoutes.js:7]: Please, fix merge conflicts with main brunch

File: src/components/Routes/ClientRoutes.js [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
-    <Routes>
-      <Route path="/" exact element={<Wallet />} />
-      <Route path="/transfer-status" exact element={<TransferStatus />} />
-    </Routes>
-  );
```
Comment (OlhaD) [inline src/components/Routes/ClientRoutes.js:12]: Also, if you use Visual Studio Code, please enable 'Prettier' as a code formatter for this project so we all have the same code layout.

File: src/components/Routes/ClientRoutes.js [hunk 3]
```diff
@@ -1,15 +1,29 @@
...
+	return (
+		<Routes>
+			<Route
+				path="/"
```
Comment (OlhaD) [inline src/components/Routes/ClientRoutes.js:11]: Am I right that the side menu is collapsed by default only in mobile view? In the desktop version, we have plantly of space so I think it should display in full size.

File: src/components/layout/Layout.js [hunk 4]
```diff
@@ -1,44 +1,40 @@
...
+	const handleDrawerClose = () => {
+		setOpen(false);
+	};
+
+	let location = useLocation();
+
+	useEffect(() => {
+		handleDrawerClose();
+	}, [location]);
+
```
Comment (OlhaD) [inline src/components/layout/Layout.js:23]: I don't think we need this functionality n the desktop version as we have plantly of space and we can show the menu with text.

File: src/components/layout/Layout.js [hunk 5]
```diff
@@ -1,44 +1,40 @@
...
+				handleDrawerOpen={handleDrawerOpen}
+			/>
+			<StyledContent
+				container
+				onClick={handleDrawerClose}>
+				{children}
+			</StyledContent>
+		</Box>
+	);
 };
```
Comment (OlhaD) [inline src/components/layout/Layout.js:38]: This is how I see the main page layout: 
Screenshot (51)
Page content should fit the page and user should not see the vertical scroll.

Comment (OlhaD) [reply src/components/layout/Layout.js:38]: The same with a mobile view: 
Screenshot (52)

File: src/components/Routes/ClientRoutes.js (hunk 2 above)
Comment (garenpham) [reply src/components/Routes/ClientRoutes.js:12]: Sorry, I was using custom settings for my Prettier. I have changed the settings to match this project now.

Comment (OlhaD) [reply src/components/Routes/ClientRoutes.js:12]: No problem, thank you for changing. I will add this to the README later.

File: src/components/Routes/ClientRoutes.js (hunk 1 above)
Comment (garenpham) [reply src/components/Routes/ClientRoutes.js:7]: I have added a "Send Tokens" route, and also a test function for that route on ClientRoutes.test.js. Everything else I kept the same.

File: src/components/layout/Layout.js (hunk 5 above)
Comment (garenpham) [reply src/components/layout/Layout.js:51]: I have removed those empty white spaces on my lastest git commit

**Review by OlhaD — APPROVED** [review]

**Pull request merged**